src/index.d.ts

```diff
@@ -173,7 +173,7 @@ export interface Options {
 	traceparent?: string | null;
 }
 
-export const create: (name: string, options: Options) => Tracer;
+export const create: (name: string, options: Options, start_offset?: number) => Tracer;
```

Comment:
Hey thank you so much for the PR <3 — sorry it took a while to respond.

Do you think it might make better sense to accept an optional `Clock` interface? Something like;

```ts
type ClockLike = { now(): number };

export const create: (name: string, options: Options, clock?: ClockLike) => Tracer;

// As per your requirement:
create(..., { now() { return Date.now() + 1 }});

// Maybe in node folk can use:
import { performance } from 'perf_hooks';

create(..., performance);
```

In addition to this, ive been thinking more about leveraging more environment specifics, so heuristics report correctly in those tooling:

like `performance.measure` or `console.time` etc. 

But anyway, let me know what you think about that clock approach.

Reply:
delegating time to another entity controllable by the end user if needed is a very sound idea 👍

Reply:
Looks like a sound solution, will implement this then!